lib/stopwatch/timer.ex

```diff
@@ -33,10 +33,13 @@ defmodule Stopwatch.Timer do
     notify()
   end
 
+  # coveralls-ignore-start
```

Comment:
Using GenServer instead of Agent, so this file is not use anymore hence the subscribe function not being called. I don't want to delete the Agent so we can compare both version but I don't want to try to write a specific test for this line either